### ISSUE TITLE
SUMO to OWL translation

### ISSUE THREAD

**Issue by arademaker** [ontologyportal/sumo#180]
Is there any simple way to call the SUMO to OWL transformation from the command line? I was expecting something similar to:

```
java -Xmx7g -classpath $SIGMA_CP com.articulate.sigma.trans.SUMOKBtoTPTPKB
```

But the main method from the `com.articulate.sigma.trans.OWLtranslator` class contains only 

```java
        try {
        	OWLtranslator.read(args[0]);
        }
        catch (Exception e ) {
            System.out.println(e.getMessage());
            e.printStackTrace();
        }    	
```

The remain comments look like tests. There is also a `public void writeKB(String path) throws IOException` but it contains a strange call to `readYAGOSUMOMappings()`. Why the conversion of SUMO to OWL would depend on YAGO mapping? 

BTW, do we have any documentation about the algorithm to translate SUMO to OWL?

**Comment by apease** [comment]
I'll try to add the command line code in the next few days.  There's no documentation on the translation but it's pretty simple given how simple OWL is - just binary relations get translated with some attempt to use built-in OWL equivalents when they exist

**Comment by arademaker** [comment]
Hum, the same file contains also methods to read OWL and transform to KIF... But without documentation is hard to follow the code and understand what is going on...

**Comment by apease** [comment]
I've fixed this and added a command line interface.  You can now perform a translation with
 
java -Xmx7g -classpath /home/apease/workspace/sigmakee/build/classes:/home/apease/workspace/sigmakee/build/lib/*:/home/apease/workspace/sigmakee/lib/* com.articulate.sigma.trans.OWLtranslator -s

(changing the paths to conform to your system of course)

**Comment by arademaker** [comment]
Minor comment. The TPTP and TF0 translations save the produced file in the sumo directory. The current OWL code saves the produced file in the current directory. Not important , but may cause confusion in the future...
